Spigot-Server-Patches/0392-Avoid-hopper-searches-if-there-are-no-items.patch

```diff
@@ -74,7 +74,7 @@ index 06b4dc628..37b12daff 100644
  
 +            // Paper start - Don't search for inventories if we have none, and that is all we want
 +            /*
-+             * We check if they want inventories by seeing if it is the static `IEntitySelector.c`
++             * We check if they want inventories by seeing if it is the static `IEntitySelector.d`
```

Comment:
every time an obfuscated methods/field is used, a puppy dies; refer to its obfhelper (`isInventory()`)

Reply:
This comment is pointing towards an obfhelper where the helper has been updated, but the "what this should be" is outdated, it's technically a non-issue due to the nature of the full comment, but, there is no reason why it shouldn't be resolved